setup.cfg

```diff
@@ -46,8 +46,8 @@ version = attr: iris.__version__
 include_package_data = True
 install_requires =
     cartopy>=0.20
-    cf-units>=3
-    cftime>=1.5.0
+    cf-units==3.0.1.post0
```

Comment:
However, this version is available on PyPI and installable using `pip`... so let's keep that honest.

Reply:
The subtle thing here is that the `3.0.1` and `3.0.1.post0` PyPI wheels were built with different mechanisms... so I'd need to check what the actual differences were to roll it back to `3.0.1`.

Worth understanding... I'll check

Reply:
Ahhh okay... so the PyPI `3.0.1` only contains the `sdist` with no `wheels`.

Whereas `3.0.1.post0` has both i.e., the `post0` change was the first iteration of infrastructure on `cf-units` to automate and build the `wheels` for `py37`, `py38` and `py39` from the `sdist`.

Reply:
`pip` can install from `sdist`, although it's less preferred over using a `wheel`.

Let me experiment as see if it's safe to roll back to `3.0.1` to keep things consistent.

Reply:
Okay, so my recommendation is to keep `site.cfg` with `cf-units==3.0.1.post0`. Why?

Simply because the `3.0.1.post0` version from PyPI will install with a `wheel`, which includes the required `udunits2` shared object that `cf-units` links into at runtime i.e., if you `pip install cf-units==3.0.1.post0` it will "just work" :tm:

If users `pip install scitools-iris==3.2.1.post0` for whatever reason and we pin `cf-units==3.0.1` within `setup.cfg`, then it could hit the problems that `cf-units==3.0.1.post0` solves by providing bdist wheels i.e., installing `cf-units` from a `sdist` relies on the environment already having `udunits2` available, which is not guaranteed to be the case, and if so the install with fail.

What say ye @trexfeathers and @lbdreyer ?

Reply:
OK, thanks for explaining 👌